test/test_sid/test-5-expected-ietf-constrained-voucher@2019-08-01.sid

```diff
@@ -1,76 +1,76 @@
 {
   "assignment-ranges": [
     {
-      "entry-point": 2500,
+      "entry-point": 2500, 
```

Comment:
Why is there a space character in the end?

Reply:
The file is generated with json.dumps(), I don't have the control on spaces. It seems that the behavior of this Lib changed between the generation of the original files and now.